README.md

```diff
@@ -0,0 +1,121 @@
+# LegacyClonk
+
+![Logo](planet/Graphics.c4g/Logo.png)
+
+LC (short for LegacyClonk) is a fan project based on Clonk Rage.
+LegacyClonk continues to receive updates and ensures compatibility with existing Clonk Rage content.
+
+The goal is to fix as many bugs and inelegances as possible, improving quality-of-life and adding nice-to-have features while not impacting the gameplay we’re all used to.
+
+## Installation
+Please refer to the [English](https://clonkspot.org/lc-en#installation-1) or [German](https://clonkspot.org/lc#installation-1) installation manual if you simply want to play the game.
+
+[clonkspot.org](https://clonkspot.org) hosts the community forum, content developer documentation and masterserver.
+
+## Compiling - Quick Start
+**Disclaimer**: This readme is fairly new and may contain mistakes. In case of problems [reach out](#Contact). Readme improvement PRs are welcome.
+
+Essential dependencies:
+- CMake
+- Fairly modern C++ compiler
+	- Windows: **Latest MSVC** (mingw-w64 g++ ≥ 13.1 *should* work with cmake option `-DUSE_WINDOWS_RUNTIME=OFF`, but is *not tested regularly*)
+	- Linux: **g++ ≥ 13.1** or **clang++ ≥ 18.1**
+	- macOS: **open source clang++ ≥ 18.1** - e.g. `brew install llvm@18 ninja`
+- Make or Ninja
+
+Extract the [latest pre-built dependencies](https://github.com/legacyclonk/deps/releases/latest) for your platform into a folder called `deps`.
+Make sure that besides the `CMakeLists.txt` of LegacyClonk there are folders `deps/include`, `deps/lib`, etc. as extracted from the binary package.
+
+
+### Configuring and compiling the engine
+Open a Terminal and navigate to the repository root, such that `CMakeLists.txt` is in the current directory.
+
+#### Windows
+Use the MSVC developer PowerShell.
+
+Configure with CMake
+```PowerShell
+cmake . -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo
+```
+
+Open the resulting Visual Studio solution or simply compile with CMake as follows
+```PowerShell
+cmake --build build
+```
```

Comment:
Visual Studio can open CMake projects directly; anyone who doesn't want that already knows how to use the terminal.